src/Adapters/Auth/facebookaccountkit.js

```diff
@@ -0,0 +1,73 @@
+const crypto = require('crypto');
+const https = require('https');
+const Parse  = require('parse/node').Parse;
+
+const graphRequest = (path) => {
+  return new Promise((resolve, reject) => {
+    https.get(`https://graph.accountkit.com/v1.1/${path}`, (res) => {
+      var data = '';
+      res.on('data', (chunk) => {
+        data += chunk;
+      });
+      res.on('end', () => {
+        try {
+          data = JSON.parse(data);
+          if (data.error) {
+            // when something wrong with fb graph request (token corrupted etc.)
+            // instead of network issue
+            reject(data.error);
+          } else {
+            resolve(data);
+          }
+        } catch (e) {
+          reject(e);
+        }
+      });
+    }).on('error', function () {
+      reject('Failed to validate this access token with Facebook.');
```

Comment:
You'll want to customize this error message as it's identical to the one for facebook auth.